boxoffice/views/order.py

```diff
@@ -336,15 +336,21 @@ def edit_invoice_details(order):
     invoice_dict = request.json.get('invoice')
     if not request.json or not invoice_dict:
         return api_error(message=_(u"Missing invoice details"), status_code=400)
+
+    invoice = Invoice.query.get(request.json.get('invoice_id'))
+    if invoice.is_final:
+        return api_error(message=_(u"This invoice has already been generated and hence cannot be modified"),
```

Comment:
Shouldn't you say "…has been finalised and hence cannot be modified?" The "generated" term is ambiguous.